src/components/Header/index.js

```diff
@@ -49,13 +69,36 @@ module.exports = class Header extends React.Component {
 
         return (
 
-            <div>
-                <h1>Strangeluv</h1>
-                <NavLink exact to='/' activeClassName={Classes.activeRoute}>Home</NavLink>
-                {' · '}
-                <NavLink to='/dashboard' activeClassName={Classes.activeRoute}>Dashboard</NavLink>
-                {renderedNav}
-            </div>
+            <AppBar
+                style={{ position: 'relative' }}
+                color='secondary'
+            >
+                <Toolbar>
+                    <Typography
+                        noWrap
+                        variant='title'
+                        color='inherit'
+                        style={{ flex: 1 }}
```

Comment:
There are a few instances in this file where I used inline styles for a one-style adjustment. It felt like overkill to create a styled-component for these, but I can definitely do that if it makes the most sense.

Reply:
yeah, overall i kinda struggle on when and why to use styled-components over other means....sometimes it feels like overkill when it's a quick, one-time inline fix for a specific component. i also kinda miss just writing good ol' fashioned scss...

Reply:
Poor scss is dead for us loLL

Reply:
we're saying scss is dead, but we have a header stylesheet to manage `activeClassName` styles on nav items. Since we're also using inline styles on different header components, should we just toss that in the stylesheet so at least everything lives in one place? or do something different with active route styling and remove the header stylesheet? This is why i struggle...

Reply:
In the single `scss` file or near the component I mean

Reply:
I'd look in a scss file at the component level for stuff like that